### PR TITLE
[Swift] Added self to Swift maintainers.md / added six concept exercises

### PR DESCRIPTION
**Pull request by wneumann**
Also added config.json file with no exercises yet.

### CODE REVIEW

File: languages/swift/README.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ This area will contain everything needed to launch the Swift track, including:
 Before we publicize requesting contribution for this language, the following steps should be done.
 
 - [ ] Have a kick-off discussion between track maintainers
-- [ ] Fill out the [maintainers.md](./maintainers.md) file (e.g. [C#](../csharp/maintainers.md))
+- [x] Fill out the [maintainers.md](./maintainers.md) file (e.g. [C#](../csharp/maintainers.md))
 - [x] Ensure there is a link to your track's GitHub issues on the [main README.md](../../README.md)
-- [ ] [Write a Concept Exercise implementation guide](../../docs/maintainers/writing-a-concept-exercise-github-issue.md)
+- [x] [Write a Concept Exercise implementation guide](../../docs/maintainers/writing-a-concept-exercise-github-issue.md)
```
Comment (ErikSchierboom) [inline languages/swift/README.md:18]: I don't think this is correct. If I look at [that document](https://github.com/exercism/v3/blob/master/languages/swift/reference/implementing-a-concept-exercise.md), it is still the default (stub) implementation. For an example of what that file can look like, see https://github.com/exercism/v3/blob/master/languages/csharp/reference/implementing-a-concept-exercise.md

```suggestion
- [ ] [Write a Concept Exercise implementation guide](../../docs/maintainers/writing-a-concept-exercise-github-issue.md)
```

**Comment by wneumann** [comment]
This turned into a huge PR as I added the scaffolding for exercises plus six concept exercises. I apologize for that. Hopefully I can learn to work git/github better to avoid this in the future.

**Comment by ErikSchierboom** [comment]
> This turned into a huge PR as I added the scaffolding for exercises plus six concept exercises. I apologize for that. Hopefully I can learn to work git/github better to avoid this in the future.

No worries. I am thinking that maybe we should try to fix this before reviewing. Could you perhaps split this PR into multiple PR's? If you need any help with how to do that, please let me know!

**Comment by iHiD** [comment]
Awesome that you've done 6 Concept Exercises :)

**Comment by wneumann** [comment]
Yes, I will definitely need to devote some time to learning the ways of GitHub. I was somewhat surprised when I pushed my stuff to my fork and it all ended up in this old PR. I can assure you future ones will be smaller and easier to digest.

**Comment by ErikSchierboom** [comment]
Awesome! And once again thanks for working on the Swift track!

**Pull request merged**